### PR TITLE
fix: mark transformed checks as deleted

### PR DESCRIPTION
**Pull request by yashmehrotra**
Fixes: https://github.com/flanksource/canary-checker/issues/1093

### CODE REVIEW

File: pkg/db/canary.go [hunk 1]
```diff
@@ -109,7 +109,10 @@ func UpdateChecksStatus(ids []string, status models.CheckHealthStatus) error {
 	}
 	return Gorm.Table("checks").
 		Where("id in (?)", ids).
-		Update("status", status).
+		Updates(map[string]any{
+			"status":     status,
+			"deleted_at": time.Now(),
```
Comment (moshloop) [inline pkg/db/canary.go:114]: shouldnt this be server now ?

Comment (yashmehrotra) [reply pkg/db/canary.go:114]: time.Now() also has tz context, so it should be the same

Comment (moshloop) [reply pkg/db/canary.go:114]: Not if there is a clock skew between db and server

Comment (yashmehrotra) [reply pkg/db/canary.go]: 👍🏽

**Pull request merged**